internal/server/conf.go

```diff
@@ -16,11 +16,14 @@ import (
 )
 
 const (
-	confKey                     = "server"
-	defaultHTTPListenAddr       = ":3592"
-	defaultGRPCListenAddr       = ":3593"
-	defaultAdminUsername        = "cerbos"
-	defaultRawAdminPasswordHash = "$2y$10$VlPwcwpgcGZ5KjTaN1Pzk.vpFiQVG6F2cSWzQa9RtrNo3IacbzsEi" //nolint:gosec
+	confKey                       = "server"
+	defaultHTTPListenAddr         = ":3592"
+	defaultGRPCListenAddr         = ":3593"
+	defaultAdminUsername          = "cerbos"
+	defaultRawAdminPasswordHash   = "$2y$10$VlPwcwpgcGZ5KjTaN1Pzk.vpFiQVG6F2cSWzQa9RtrNo3IacbzsEi" //nolint:gosec
+	defaultMaxActionsPerResource  = 50
+	defaultMaxResourcesPerRequest = 50
+	requestItemsMax               = 500
```

Comment:
I'm not sure we really need to enforce this limit - e.g. if we'd allow 500×500, why wouldn't we allow 1000×10? Once you get into tuning these away from the defaults I think you're in the realm of "at your own risk", and it's clearly documented that it can affect performance.

Reply:
Well, it's at the user's risk but they could have made a typo or made the change without really understanding the implications ("just stick a large number in" mentality) that could come back to bite them later. I feel that as we are more intimately familiar with the code, we should provide some guard rails here.

500x500 is different from 250x1000 or 10000x25 because the number of actions determine how many times the rules in a policy get evaluated and the number of resources determine how many policies need to be loaded from storage. So they all have different performance characteristics and that's why I chose to expose these two numbers rather than just their multiplication.